etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -0,0 +1,250 @@
+import pathlib
+import tempfile
+import zipfile
+from typing import Optional
+from typing import Union
+
+import numpy as np
+import pandas as pd
+from typing_extensions import Literal
+
+from etna.libs.ts2vec import TS2Vec
+from etna.transforms.embeddings.models import BaseEmbeddingModel
+
+
+class TS2VecEmbeddingModel(BaseEmbeddingModel):
+    """TS2Vec embedding model."""
```

Comment:
We should probably reference some paper about this model to allow users to read how it works.